### ISSUE TITLE
Endless Scrolling

### ISSUE THREAD

**Issue by vrindavan** [TeamNewPipe/NewPipe#5525]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Endless Scrolling... currently it stops after few videos, instead it could be better if we can have endless videos below and once a user starts scrolling...the current video shrinks and displays a horizontal slick banner for the user to get back to the top by clicking it.

**Comment by mhmdanas** [comment]
Can you clarify? Where do you want endless scrolling exactly?

**Comment by vrindavan** [comment]
Play any video in youtube android or vanced, then when you scroll down you get video related to that, right? On original YT we get lots of videos on scrolling, it does end after a few thought but for normal users it is almost endless. On NP this is even less, lot less so my suggestion was to make it more or endless (video suggestions below the Now Playing). When the user scrolls in original YouTube app the currently playing video stays there so if it is possible then you can add something like that.

**Comment by SameenAhnaf** [comment]
@mhmdanas Close this as #6156 has a far better explanation.

**Comment by SameenAhnaf** [comment]
In my opinion, 25 videos should load at a time. New videos should load only if a user scrolls down to the bottom. Loading videos continuously is useless unless a user scrolls down.

**Comment by vrindavan** [comment]
Yes that's good enough. Because unless the user scrolls to the very bottom, it's not necessary to put more videos.

I think not even 25, but 15 or 20 at startup would do.
